examples/hpc-enterprise-slurm.yaml

```diff
@@ -0,0 +1,233 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-enterprise-slurm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hpc01
+  region: us-central1
+  zone: us-central1-a
+  gpu_zones: [us-central1-a, us-central1-b, us-central1-c, us-central1-f]
+  # Visit https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#published-image-family
+  # for a list of valid family options with Slurm
+  family: schedmd-v5-slurm-22-05-9-hpc-centos-7
+  project: projects/schedmd-slurm-public/global/images/family
```

Comment:
Yes, it hopefully allows us to parametrize this in the integration tests and test various images. Otherwise, it is hard to parametrize a larger data structure.